Build/Build.sln

```diff
@@ -0,0 +1,51 @@
+﻿
```

Comment:
As you brought up in the teams this has an unfortunate issue with asset linking to code by script reference.

We may be able to use the workflow where the code lives outside of Unity and built to DLLs and placed in the Unity project if we can get debugging to work more easily.

Reply:
Seems like this scheme is fundamentally broken without fixing the script issue. Is that ok for this PR since you're just moving changes into the feature branch?

Reply:
That's the plan for now. Feature branch won't merge until we've fixed issues.